tests/lit-tests/vectorcall-5.ispc

```diff
@@ -1,5 +1,6 @@
 // This test checks function signature of externs with '__vectorcall' to ensure it has 'x86_vectorcallcc'.
-// RUN: %{ispc}  %s --emit-llvm-text --target=host -o - | FileCheck %s
+// RUN: %{ispc}  %s --emit-llvm-text --vectorcall --target=host -o - | FileCheck %s
+// RUN: %{ispc}  %s --emit-llvm-text --no-vectorcall --target=host -o - | FileCheck %s
```

Comment:
This fails for the --no-vectorcall option because check looks for vectorcall

Reply:
I've changed the test, we'll need to change behavior later.